webview-ui/src/utils/platformUtils.ts

```diff
@@ -34,3 +34,9 @@ export const detectMetaKeyChar = (platform: string) => {
 		return "CMD"
 	}
 }
+
+const userAgent = navigator.userAgent
+
+export const isChrome = userAgent.indexOf("Chrome") >= 0
```

Comment:
This may be undefined, and can throw an error so i would want to be safer about this like `navigator && navigator.userAgent && navigator.userAgent.indexOf("Chrome") >= 0`